torch/_dynamo/variables/tensor.py

```diff
@@ -924,6 +924,8 @@ def call_method(
         if name in ["__len__", "size", "tolist"]:
             # delegate back to TensorVariable
             return super().call_method(tx, name, args, kwargs)
+        if name == "tobytes":
+            unimplemented("tobytes is not modelled in torch._numpy")
```

Comment:
On a second thought, this would have been best implemented in the translation layer by raising a NotImplementedError, but well